lib/whitehall_importer/import.rb

```diff
@@ -73,5 +74,16 @@ def first_publish_date
     def first_edition_history
       EditionHistory.new(whitehall_document["editions"].first["revision_history"])
     end
+
+    def integrity_check(document)
+      integrity_check_edition(document.live_edition) if document.live_edition
+      integrity_check_edition(document.current_edition) if document.current_edition != document.live_edition
+    end
+
+    def integrity_check_edition(edition)
```

Comment:
These two methods may feel more consistent with the methods with the rest of this file if they start with the verb. For example `check_document_integrity` and `check_edition_integrity`